forte/data/readers/tests/corpus_reader_test.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2019 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Tests for corpus_reader.
+"""
+import os
+import unittest
+from typing import Dict
+
+from ft.onto.base_ontology import Document
+
+from forte.data.data_pack import DataPack
+from forte.data.readers.corpus_reader import CorpusReader
+
+
+class CorpusReaderTest(unittest.TestCase):
+    def setUp(self):
+        self.reader = CorpusReader()
```

Comment:
I just realized that ``CorpusReader`` isn't a good name, cuz I cannot recall what this test is about. Could you change it to a more informative name?